consutil/main.py

```diff
@@ -73,6 +74,10 @@ def clear(db, target, devicename):
 @click.option('--devicename', '-d', is_flag=True, help="connect by name - if flag is set, interpret linenum as device name instead")
 def connect(db, target, devicename):
     """Connect to switch via console device - TARGET is line number or device name of switch"""
+    if os.geteuid() != 0:
```

Comment:
I will raise another PR to remove this because we need a udev rule update in sonic-buildimage before doing that.